src/articleList.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+(function(exports) {
+function ArticleList() {
+  this._articles = [];
+};
+
+ArticleList.prototype = {
+  save: function(headline, body, url, date, thumbImage, mainImage) {
+    var article = new Article(headline, body, url, date, thumbImage, mainImage);
+    this._articles.push(article);
+    var idnum = this._articles.length
+    article._id = idnum;
+    return this._articles;
+  },
+
+  articles: function() {
+    return this._articles;
+  },
+
+  getArticleById: function(id) {
+    for(var i = 0; i < this._articles.length; i++) {
```

Comment:
You could use _articles.find() here - see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/prototype for a full list of collection methods